modules/scripts/startup-script/files/startup-script-stdlib-head.sh

```diff
@@ -30,6 +30,45 @@ readonly E_RUN_OR_DIE=5
 readonly E_MISSING_MANDATORY_ARG=9
 readonly E_UNKNOWN_ARG=10
 
+SUCCESS_MESSAGE=$(
+	cat <<-EOF
+		** NOTICE **: The VM startup scripts have finished running successfully.  
+		It is now safe to start using the system.
+	EOF
+)
+readonly SUCCESS_MESSAGE
```

Comment:
is it possible to include this in the declaration? It looks funny coming so many lines later.

Reply:
I think the issue was that it didn't like the command `cat<<-EOF` with the assignment and readonly.  This was the solution I found after some research.